corehq/pillows/mappings/group_mapping.py

```diff
@@ -45,9 +45,8 @@
         "name": {
             "fields": {
                 "exact": {
-                    "index": "analyzed",
+                    "index": "not_analyzed",
                     "type": "string",
```

Comment:
I noticed when looking at the docs that ES7 no longer has a "string" type, and that it's been replaced with "text" and "keyword" types. Does that mean that it's impossible to use a single mapping for ES2 and ES7? Or does ES7 have backwards compatibility for `{"type": "string", "index": "not_analyzed"}` to be interpreted as `{"type": "keyword"}`?